modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -14,17 +14,20 @@ def perform(auto_claim_id)
       form_data = auto_claim.to_internal
       auth_headers = auto_claim.auth_headers
 
-      begin
-        response = service(auth_headers).submit_form526(form_data)
-        auto_claim.evss_id = response.claim_id
-        auto_claim.status = ClaimsApi::AutoEstablishedClaim::ESTABLISHED
-        auto_claim.save
-      rescue ::EVSS::DisabilityCompensationForm::ServiceException => e
-        auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-        auto_claim.evss_response = e.messages
-        auto_claim.save
-        raise e
-      end
+      response = service(auth_headers).submit_form526(form_data)
+      auto_claim.evss_id = response.claim_id
+      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ESTABLISHED
+      auto_claim.save
+    rescue ::EVSS::DisabilityCompensationForm::ServiceException => e
+      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
+      auto_claim.evss_response = e.messages
+      auto_claim.save
+      raise e
+    rescue ::Common::Exceptions::BackendServiceException => e
+      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
+      auto_claim.evss_response = [{ 'key' => e.status_code, 'severity' => 'FATAL', 'text' => e.original_body }]
+      auto_claim.save
+      raise e
```

Comment:
is there a way to just add in the sentry logging here (and in the other rescue block) so that way it doesn't get retried, like we were talking through before?